### PR TITLE
Feature/add service test

### PR DESCRIPTION
**Pull request by garyluu**
Adds a new service for testing via generateChangeLog + diff (hopefully there is some better way).  

Uses the 1.7.0-beta.3 webservice

### CODE REVIEW

File: travisci/add_service.xml [hunk 1]
```diff
@@ -0,0 +1,409 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
```
Comment (denis-yuen) [inline travisci/add_service.xml]: Move this to dockstore-webservice/src/main/resources like dockstore-webservice/src/main/resources/migrations.test.samepaths.xml to keep things organized

Change the name to add_service_1.7.0.xml ala our confidential migrations so we know what version this was for

**Comment by garyluu** [comment]
> Make sure that any docs (wiki) that reference how to do migrations includes this new file and the purpose of it.

This migration file is only used for tests and nowhere else.  CircleCI config.yaml has the commands needed.  Should be obvious

File: cypress/integration/immutableDatabaseTests/services.ts [hunk 2]
```diff
@@ -32,19 +32,19 @@ describe('Dockstore Home', () => {
     });
   });
 
-  // TODO: This is a fake service, need to use a more realistic one later
   describe('services', () => {
-    it('Have no services in /services', () => {
+    it('Have one services in /services', () => {
```
Comment (coverbeck) [inline cypress/integration/immutableDatabaseTests/services.ts]: service not services

File: scripts/run-webservice-script.sh [hunk 3]
```diff
@@ -10,5 +10,5 @@ psql -h localhost -c "create user dockstore with password 'dockstore' createdb;"
 psql -h localhost -c "ALTER USER dockstore WITH superuser;" -U postgres
 psql -h localhost -c 'create database webservice_test with owner = dockstore;' -U postgres
 psql -h localhost -f travisci/db_dump.sql webservice_test -U postgres
-java -jar dockstore-webservice.jar db migrate -i 1.5.0,1.6.0,1.7.0 travisci/web.yml
+java -jar dockstore-webservice.jar db migrate -i 1.5.0,1.6.0,1.7.0,add_service_1.7.0 travisci/web.yml
```
Comment (denis-yuen) [inline scripts/run-webservice-script.sh]: Just double checking, but won't this add the test service to all tests whether they're testing services or not?

Comment (garyluu) [reply scripts/run-webservice-script.sh]: ya, but it shouldn't really be a problem.  Our initial DB also adds tools/workflows for all tests

Comment (denis-yuen) [reply scripts/run-webservice-script.sh]: Ok, just making sure it was intentional

**Pull request merged**